app/api/api_v1/endpoints/projects.py

```diff
@@ -328,3 +332,68 @@ def __persist_pics_update(db: Session, project: Project) -> Project:
     db.commit()
     db.refresh(project)
     return project
+
+
+@router.get("/{id}/export", response_model=schemas.ProjectCreate)
+def export_project_config(
+    *,
+    db: Session = Depends(get_db),
+    id: int,
+    download: bool = False,
+) -> JSONResponse:
+    """Retrive the project config by id.
+
+    Args:
+        id (int): project id
+
+    Raises:
+        HTTPException: if no project exists for provided project id
+
+    Returns:
+        Project: project The project by the informed project id
```

Comment:
"project The project" is a little odd. Also, I suggest adding the return type, something like this:

```suggestion
        JSONResponse: json representation of the project with the informed project id
```